### DISCUSSION TITLE
Adjustable oxygen generator output with scaling power usage.

### DISCUSSION THREAD

**Discussion by link12313** [FakeFishGames/Barotrauma#5346] [Ideas and suggestions]
The oxygen generator in it's current form is a bit basic and a power hog, What I'm requesting is a setting that drops it's output and power usage linearly. A key use case for this would be when most of a subs crew is on EVA and the sub is idling above a point of interest. You could run the rods cold and set the oxygen generator to low to save a huge amount of power in the more power hungry subs. Also the vents could be on a toggle so that lower output would be sent only to crewed rooms in that mode.

**Comment by diegoyocys** [comment]
Seems like a good idea,  but i dont know about the toggle vents, seems like ppl would completely forget and die! Lol

You could Also potentially unplug the oxigenator? The existing oxigen quality should decay slowly, and yall should have a diving mask, just in case

**Reply by link12313** [reply]
That's a bit too risky on single player. The vent toggle would be controlled by wires along with the oxygen output as well as having a UI for manual control.
